app/src/main/java/com/teamagam/gimelgimel/app/map/esri/graphic/EsriSymbolCreator.java

```diff
@@ -0,0 +1,40 @@
+package com.teamagam.gimelgimel.app.map.esri.graphic;
+
+
+import android.content.Context;
+
+import com.esri.core.symbol.Symbol;
+
+public class EsriSymbolCreator {
+
+    private Context mContext;
+
+    public EsriSymbolCreator(Context context) {
+        mContext = context;
+    }
+
+    public Symbol create(com.teamagam.gimelgimel.domain.map.entities.symbols.Symbol ggSymbol) {
+        Symbol specificSymbol = transform(ggSymbol);
```

Comment:
I know I named it specificSymbol, but I thought about constructing the method a little different.
This name at the current structure seems a little weird, what does the "specific" mean? The same variable gets the "selection" symbology aswell.